These tests pass most of the time, but they intermittently fail rarely,
which could cause a problem for passing CI.

--- a/tests/arena.impure.lisp
+++ b/tests/arena.impure.lisp
@@ -590,7 +590,7 @@
            (string (sb-vm:with-arena (a) (read-line stream))))
       string)))
 
-(test-util:with-test (:name :read-line)
+(test-util:with-test (:name :read-line :skipped-on :android)
   (test-read-line)
   (assert (null (c-find-heap->arena))))
 
--- a/tests/stream.impure.lisp
+++ b/tests/stream.impure.lisp
@@ -697,7 +697,7 @@
 ;; We used to not return from read on a named pipe unless the external-format
 ;; routine had filled an input buffer. Now we'll return as soon as a request
 ;; is satisfied, or on EOF. (https://bugs.launchpad.net/sbcl/+bug/643686)
-#-win32
+#-(or win32 android)
 (with-test (:name :overeager-character-buffering :skipped-on :win32)
   (let ((use-threads #+sb-thread t)
         (proc nil)
